UI/Selenium/Steps/ConsultationRoomSteps.cs

```diff
@@ -110,5 +110,75 @@ public void ThenAllParticipantsLeaveConsultationRoom()
                 Driver.FindElement(ParticipantWaitingRoomPage.JoinPrivateMeetingButton).Displayed.Should().BeTrue();
             }
         }
+
+        [Then(@"both participants click on the leave button to leave the room")]
+        public void ThenBothParticipantsLeaveConsultationRoom()
+        {
+            var participants = _hearing.Participant.Where(p => p.Party.Name.ToLower().Contains("claimant") || p.Party.Name.ToLower().Contains("defendant")).ToList();
```

Comment:
We need to talk about this, I want to be sure what kind of participant this can be, I think that using just claimant and defendant will not work in many cases, what it MIGHT need to be is anyone that is not judiciary staff.  So this would need to support other roles such as "McKenzie Friend" which you might not be aware of.